### PR TITLE
Add initial solution code

### PR DESCRIPTION
**Pull request by sethlopez-bnr**
Upgraded the gradle version to the latest.
Renamed the app to match the related codelab.
Added the solution code for the codelab.
Removed the Work in progress status from the README.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -15,5 +15,5 @@
   -->
 
 <resources>
-    <string name="app_name">Birthday Card</string>
+    <string name="app_name">Happy Birthday</string>
 </resources>
```
Comment (KeenWarrior) [inline app/src/main/res/values/strings.xml]: New line at eof

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -23,12 +23,28 @@
...
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="16dp"
+        android:layout_marginBottom="16dp"
+        android:fontFamily="sans-serif-light"
+        android:text="From Emma."
+        android:textColor="@android:color/black"
+        android:textSize="34sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent" />
 </androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (KeenWarrior) [inline app/src/main/res/layout/activity_main.xml]: New line at eof

**Comment by KeenWarrior** [comment]
For CLA. Take a look at https://github.com/google-developer-training/basic-android-java-birthday-card-app/blob/main/CONTRIBUTING.md

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Birthday Card app - Solution Code [Work in progress 🚧]
```
Comment (KeenWarrior) [inline README.md:1]: Let's not remove it till all branches are ready.

**Comment by KeenWarrior** [comment]
As per standup discussion. Let's put complete app in main branch and create separate branch for individual codelab checkpoints.

**Comment by sethlopez-bnr** [comment]
@KeenWarrior I've pushed the branches `starting-point-for-codelab-2` and `starting-point-for-codelab-3`. 👍

**Pull request merged**